packages/atomic/cypress/integration/facets/category-facet/category-facet-actions.ts

```diff
@@ -33,9 +33,72 @@ export const addCategoryFacet =
   };
 
 export function selectChildValueAt(index: number) {
-  CategoryFacetSelectors.childValue().eq(index).click();
+  CategoryFacetSelectors.childValue()
+    .find('[part="value-label"]')
+    .eq(index)
+    .then(([childValue]) => {
+      const text = childValue.textContent!;
+      cy.wrap(childValue).click();
+      CategoryFacetSelectors.activeParentValueWithText(text).should('exist');
```

Comment:
This line is what replaces `cy.wait`. I chose to put them in assertions instead of putting them in specs, because I think it'll make it much easier to add more tests and reduces risks of us forgetting a `should` somewhere.